1.purpose_and_goals.md

```diff
@@ -22,13 +22,13 @@ Additional goals of the specification are to:
 
 Non-goals include:
 
-- Defining or prescribing specific development or operational workflows.
+- Defining or prescribing specific development or operational workflow.
```

Comment:
I believe this was more grammatically correct with the "s". "Defining or prescribing _a_ specific development or operational workflow" would also read well.